api/pom.xml

```diff
@@ -49,8 +49,8 @@
 
 		<dependency>
 			<groupId>org.openmrs.module</groupId>
-			<artifactId>inventory-api</artifactId>
-			<version>2.0.10-SNAPSHOT</version>
```

Comment:
This dependancies need to get to the main pom.xml

Reply:
Resolved